Shoko.Server/Renamer/RenameFileHelper.cs

```diff
@@ -131,11 +128,8 @@ public static (ImportFolder, string) GetDestination(SVR_VideoLocal_Place place,
                 }
                 catch (Exception e)
                 {
-                    Logger.Error($"Renamer threw an error. The offending plugin was: {renamer.GetType().GetAssemblyName()} with renamer: {renamer.GetType().Name}. The error was: {e}");
```

Comment:
We shouldn't be losing the logging detail, so I would like something back, even if it was in the `continue` case

Reply:
Added logging back in [f8c288b](https://github.com/ShokoAnime/ShokoServer/pull/920/commits/f8c288be84a0928494e53235d9d3cde32c7f09b2)